### PR TITLE
MMB: Add apple wallet screen

### PR DESCRIPTION
**Pull request by tbergq**

### CODE REVIEW

File: app/MMB/src/navigation/AppleWalletScreen.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <View style={styles.container}>
+      <AppleWalletScene />
+    </View>
+  );
+}
+
+AppleWalletScreen.navigationOptions = () => ({
+  headerStyle: {
+    backgroundColor: 'rgba(0,0,0,0)',
```
Comment (RobinCsl) [inline app/MMB/src/navigation/AppleWalletScreen.js]: Maybe `Color.black` instead?

File: app/MMB/src/scenes/appleWallet/AppleWalletScene.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    </React.Fragment>
+  );
+}
+
+const styles = StyleSheet.create({
+  content: {
+    flex: 1,
+    backgroundColor: Color.white,
+    borderTopStartRadius: 10,
+    borderTopEndRadius: 10,
```
Comment (RobinCsl) [inline app/MMB/src/scenes/appleWallet/AppleWalletScene.js]: Does `borderTopHorizontalRadius` exist by any chance? 😄

Comment (tbergq) [reply app/MMB/src/scenes/appleWallet/AppleWalletScene.js]: I wish it did 😞

Comment (tbergq) [reply app/MMB/src/scenes/appleWallet/AppleWalletScene.js]: Or just borderTopRadius would also be fine 😄

**Pull request merged**